source/core/Date.cpp

```diff
@@ -60,14 +60,6 @@ namespace Vireo {
     Int32 Date::_systemLocaleTimeZone = 0;
 
     //------------------------------------------------------------
-    Int64 SecondsFromBaseYear(Int64 year, Int64 baseYear)
```

Comment:
If this is written fairly recently and may be useful in the near future, you can just #ifdef out this for now. We can delete it once the feature is close to completion.